test/cli_misc_test.go

```diff
@@ -88,19 +88,55 @@ func TestSettingMillisWorks(t *testing.T) {
 	h.MkDir(funcName)
 	h.Cd(funcName)
 	h.WithMinimalFunctionSource()
-	h.FileAppend("func.yaml", "\ncpus: 50m\n")
+	h.FileAppend("func.yaml", "\ntimeout: 50\n\nschema_version: 20180708\n")
+	h.Fn("--verbose", "deploy", "--app", appName, "--local").AssertSuccess()
+	h.Fn("invoke", appName, funcName).AssertSuccess()
+
+	inspectRes := h.Fn("inspect", "fn", appName, funcName)
+	inspectRes.AssertSuccess()
+	if !strings.Contains(inspectRes.Stdout, `"timeout": 50`) {
+		t.Errorf("Expecting fn inspect to contain CPU %v", inspectRes)
+	}
+
+	h.Fn("create", "fn", appName, "another", "some_random_registry/"+funcName+":0.0.2").AssertSuccess()
+
+	h.Fn("invoke", appName, "another").AssertSuccess()
+}
 
+//Memory doesn't seem to get persisted/returned
+func TestSettingMemoryWorks(t *testing.T) {
+	t.Parallel()
+
+	h := testharness.Create(t)
+	defer h.Cleanup()
+	h.WithEnv("FN_REGISTRY", "some_random_registry")
+
+	appName := h.NewAppName()
+	h.Fn("create", "app", appName).AssertSuccess()
+	res := h.Fn("list", "apps")
+
+	if !strings.Contains(res.Stdout, fmt.Sprintf("%s\n", appName)) {
+		t.Fatalf("Expecting list apps to contain app name , got %v", res)
+	}
+
+	funcName := h.NewFuncName()
+
+	h.MkDir(funcName)
+	h.Cd(funcName)
+	h.WithMinimalFunctionSource()
+	h.FileAppend("func.yaml", "memory: 100\nschema_version: 20180708\n")
 	h.Fn("--verbose", "deploy", "--app", appName, "--local").AssertSuccess()
-	h.Fn("call", appName, funcName).AssertSuccess()
-	inspectRes := h.Fn("inspect", "route", appName, funcName)
+	h.Fn("invoke", appName, funcName).AssertSuccess()
+
+	inspectRes := h.Fn("inspect", "fn", appName, funcName)
 	inspectRes.AssertSuccess()
-	if !strings.Contains(inspectRes.Stdout, `"cpus": "50m"`) {
```

Comment:
i wounder if this is a regression

Reply:
good catch, we have removed it from API in fns though